bitnami/mongodb/templates/replicaset/scripts-configmap.yaml

```diff
@@ -125,4 +140,102 @@ data:
     export MONGODB_ROOT_PASSWORD="" MONGODB_USERNAME="" MONGODB_DATABASE="" MONGODB_PASSWORD=""
     export MONGODB_ROOT_PASSWORD_FILE="" MONGODB_USERNAME_FILE="" MONGODB_DATABASE_FILE="" MONGODB_PASSWORD_FILE=""
     exec /opt/bitnami/scripts/mongodb/entrypoint.sh /opt/bitnami/scripts/mongodb/run.sh
+  {{- if .Values.replicaSetConfigurationSettings.enabled }}
+  replicaSetConfigurationSettings.sh: |-
+    #!/bin/bash
+    # This script to be called when pod starts.
+    # This script sets rs settings which can not be applied via conf file
+
+    function logger ()
```

Comment:
Hey @rafariossaa, I was off last week. May I suggest that I leave the logger function as is, but remove the ($date) function as any stdout will have time stamps added by whatever log capture mechnism is in place? All I really wanted was for it to append the name of the script to any line of logging to make the stdout easier to follow (where stdout from the script is interleaved with the stdout from everything else) prepending the script name makes it clear where that logging is coming from. Otherwise a simple 'echo' would have been enough. Also, I'm not terribly keen on adding a dependancy to a simple script unless it offers good value, which is doesn't in this case, just makes it less portable and harder to debug.